images/bazelbuild/runner

```diff
@@ -42,44 +41,42 @@ cleanup_binfmt_misc() {
     ls -al /proc/sys/fs/binfmt_misc
 }
 
-# runs custom docker data root cleanup binary and debugs remaining resources
+# Runs custom docker data root cleanup binary and debugs remaining
+# resources.
 cleanup_dind() {
-    # list what images and volumes remain
     echo "Remaining docker images and volumes are:"
     docker images --all || true
     docker volume ls || true
-    # cleanup binfmt_misc
-    echo "Cleaning up binfmt_misc ..."
-    # note: we run this in a subshell so we can trace it for now
-    (set -x; cleanup_binfmt_misc || true)
+
+    # We run this in a subshell so we can trace it.
+    cleanup_binfmt_misc || true
```

Comment:
is `set -x` not required anymore

Reply:
`set -x` had been added a long time ago because we had issues with vfs, but we have not had this issue since then, so I would be enclined to disable this verbose output since it doesn't seem needed anymore.

Reply:
ok thx